cypress/e2e/cloud/buckets.test.ts

```diff
@@ -15,6 +15,74 @@ const setupData = (cy: Cypress.Chainable, enableMeasurementSchema = false) =>
       )
     )
   )
+
+const testSchemaFiles = (
+  cy: Cypress.Chainable,
+  isCsv: boolean,
+  origFileContents: string,
+  checkContents: (cy: Cypress.Chainable) => void
+) => {
+  cy.getByTestID('Create Bucket').click()
+  cy.getByTestID('bucket-form-name').type('explicit_bucket')
+  cy.getByTestID('accordion-header').click()
+  cy.getByTestID('explicit-bucket-schema-choice-ID').click()
+
+  cy.getByTestID('bucket-form-submit').click()
+
+  cy.getByTestID(`bucket-card explicit_bucket`)
+    .should('exist')
+    .within(() => {
+      cy.getByTestID('bucket-settings').click()
+    })
+  cy.getByTestID('accordion-header').click()
+
+  cy.getByTestID('measurement-schema-add-file-button').click()
+  cy.getByTestID('input-field').type('first schema file')
+
+  let schemaFile = 'valid.json'
+  let type = 'application/json'
+
+  if (isCsv) {
+    schemaFile = 'valid.csv'
+    type = 'text/csv'
+  }
+
+  const testFile = new File([origFileContents], schemaFile, {type})
+
+  const event = {dataTransfer: {files: [testFile]}, force: true}
+  cy.getByTestID('dndContainer')
+    .trigger('dragover', event)
+    .trigger('drop', event)
```

Comment:
are these reliable in cypress tests? I thought cypress had issues with these kinds of operations

Reply:
this part consistently works; and this was refactored from an existing test.

Reply:
also, the drag-n-drop to put in a file was copied from an existing test as well for csv uploads